apps/storybook/src/ExpandableBlock.test.ts

```diff
@@ -11,6 +11,7 @@ describe('ExpandableBlock', () => {
     'Small',
     'Status Icon',
     'With Caption',
+    'Disabled',
```

Comment:
for disabled block, there is no "open" state to test, so you can add an if condition around lines 22-23

Reply:
![image](https://github.com/iTwin/iTwinUI/assets/44583237/a66b4695-e7b8-4116-bcbf-a1bae21b9b6c)
Does something like this work?

Reply:
yup, that's good. just use `!==` instead of `!=`

Reply:
this image can be deleted now. https://github.com/xman343/iTwinUI/blob/xander/expandable-block-disabled-/apps/storybook/cypress-visual-screenshots/baseline/ExpandableBlock.test.ts-Disabled%20(Open).png